python/pyspark/sql/context.py

```diff
@@ -226,28 +226,34 @@ def createDataFrame(self, data, schema=None, samplingRatio=None):
         from ``data``, which should be an RDD of :class:`Row`,
         or :class:`namedtuple`, or :class:`dict`.
 
-        When ``schema`` is :class:`DataType` or datatype string, it must match the real data, or
```

Comment:
I made a mistake here, thinking "datatype string" was actually meant to be `StringType()`. I understand now that a datatype string is actually a thing.

Correction incoming...